pkg/operator/validations.go

```diff
@@ -34,3 +34,13 @@ func ValidateDurationField(durationField string) error {
 	}
 	return nil
 }
+
+func ValidateRelabelAction(relabelAction string) bool {
+	validRelabelActions := []string{"replace", "keep", "drop", "hashmod", "labelmap", "labeldrop", "labelkeep"}
+	for _, action := range validRelabelActions {
+		if action == relabelAction {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This function does not really validate anything so perhaps it would be better named as `IsValidRelabelAction`? Or potentially, instead of this function maintain a lookup `ValidRelabelActions map[string]struct{}{}`?

To me, the map would be clearer and would avoid allocating the slice each time inside the function (which could also be declared as a package level var in the current implementation)

Reply:
Good suggestion. I will update :)

Reply:
updated